instrumentation/netty/README.md

```diff
@@ -0,0 +1,35 @@
+# brave-instrumentation-netty
+
+This module contains netty http handler.
```

Comment:
no,it's not version specific.But i think 4.1.x will be better. Netty 5.0.x has been discarded.
Netty 4.0.x not tested.